daemons/execd/execd_commands.c

```diff
@@ -1295,7 +1295,8 @@ lrmd_rsc_execute_service_lib(lrmd_rsc_t * rsc, lrmd_cmd_t * cmd)
         && pcmk__str_eq(cmd->action, "stop", pcmk__str_casei)) {
 
         cmd->result.exit_status = PCMK_OCF_OK;
-        goto exec_done;
+        cmd_finalize(cmd, rsc);
+        return TRUE;
```

Comment:
By the way, this is a static function and we don't use its return code at all.

Reply:
good point

Reply:
added commit on top